src/backend/gl/src/lib.rs

```diff
@@ -671,3 +672,7 @@ fn resolve_sub_range(
     let end = sub.size.map_or(whole.end, |s| whole.start + sub.offset + s);
     whole.start + sub.offset..end
 }
+
+pub(crate) const fn is_webgl() -> bool {
```

Comment:
no need for `pub(crate)` here. Private functions are automatically visible to child modules